.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: true
 contact_links:
     - name: Ask a Question
-      url: https://github.com/mesh-for-data/the-mesh-for-data-flight-module/discussions/new?category=q-a
+      url: https://github.com/fybrik.io/arrow-flight-module/discussions/new?category=q-a
```

Comment:
Should we rename `mesh-for-data` with `fybrik` or `fybrik.io`?

Reply:
You are right -- fybrik